### PR TITLE
fix: [DHIS2-10081] make it possible to delete polygons from focus area instances

### PR DESCRIPTION
**Pull request by superskip**
The second commit is not really a part of the fix - could be made into a separate PR if that's preferred.

### CODE REVIEW

File: components/registration/registration-controller.js [hunk 1]
```diff
@@ -658,6 +658,7 @@ trackerCapture.controller('RegistrationController',
             $scope.selectedTei.orgUnit = $scope.tei.orgUnit = $scope.selectedOrgUnit.id;
             $scope.selectedTei.attributes = $scope.tei.attributes = [];
         }
+        $scope.tei.featureType = $scope.selectedTei.featureType;
         $scope.tei.geometry = $scope.selectedTei.geometry;
```
Comment (Bekkalizer) [inline components/registration/registration-controller.js:662]: Geometry is the old name for feature type... perhaps this one can be removed?

**Review by Bekkalizer — CHANGES_REQUESTED** [review]
Maybe we can remove the "geometry" property? It should not be used anywhere anymore?

File: components/registration/registration-controller.js (hunk 1 above)
Comment (superskip) [reply components/registration/registration-controller.js:662]: This geometry object looks like this: 
geometry: {
    type:  A leaflet object type ("Polygon" in this case)
    coordinates: The vertices of the polygon.
}

while `tei.featureType` is a string that should contain the same value as `geometry.type`. Thus it seems more likely that it's the other way around, that `featureType` is the old name for `geometry`. However, the server still uses `featureType` as condition for deleting existing polygons, so it seems both are needed.

**Comment by Bekkalizer** [comment]
Looks good!

**Pull request merged**